terraform/layers/base/vpc.tf

```diff
@@ -0,0 +1,29 @@
+// Get names of the available AZs
+data "aws_availability_zones" "available" {
+  state = "available"
+}
+
+locals {
+  azs  = data.aws_availability_zones.available.names
+  cidr = "10.0.0.0/16"
+}
+
+
+module "vpc" {
+  source = "terraform-aws-modules/vpc/aws"
+
+  name = "dev-vpc"
```

Comment:
This would also be better as a variable, otherwise if we moved to prod the resource would be labelled dev. Also naming resources with project and environment is good practice. E.g.
name = "${var.project_name}-${var.environment}-vpc"

or creating a local, an example would be

locals {
  resource_name_prefix = "${var.project_name}-${var.environment}"
}
and naming resources like name = "${local.resource_name_prefix}-vpc"

Might help down the line - common approaches to help quickly see what reosources belong to which project / environment